content/blog/16-17/linux.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Linux: hoe doe?"
+created_at: 02-02-2017
+description: Uitleg en tips voor iedereen die linux wil installeren.
+author: Rien
+---
+
+_Tijdens het begin van ieder schooljaar komt er een verse lading studentjes Informatica toe die in de lessen Computergebruik kennis maken met de wondere wereld van GNU/Linux. Velen onder hen worden geprikkeld door de onontgonnen mysteries van dit besturingssysteem en groeien een verlangen om de kunst van het Unix-tovenaarschap zich eigen te maken._
+
+Deze blogpost dient als een leidraad voor de dappersten onder hen: zij die het pad naar de verlossing willen bewandelen en een eigen Linux-installatie willen tot leven wekken.
+
+# Enkele tips voor je begint
+---
+
+## Probeer eerst in een virtuele machine
+Als je nog geen ervaring hebt met het installeren van Linux start je best door eens te oefenen op een VM (virtuele machine). Zo komt niet alles in één keer op je af. Zo kun je eerst het installatieproces onder de knie krijgen zonder meteen rekening te hoeven houden met eventuele problemen, dual boot opzetten en andere obstakels.
+
+## Zit je vast? RTFM!
+Read The Fucking Manual (of forum, of wiki). Kom je op een rare error terecht, weet je niet wat de volgende stap is of weet je niet waar te beginnen? Raak niet direct in paniek, ieder struikelblok is een kans om te leren. Foutmeldingen geven vaak een aanwijzing waar het probleem zou kunnen liggen, lees die dus goed. Indien dat geen verlichting brengt: zoek je specifiek probleem op. De kans is klein dat jij de eerste bent met hetzelfde probleem.
```

Comment:
- krijg je een rare error
- Raak niet direct in paniek => panikeer niet onmiddellijk?
- iedere struikelblok